src/cli/commands/why.js

```diff
@@ -219,7 +219,12 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
 
     // prepare output: populate reporter
     reporter.info(reporter.lang('whyMatch', `${matchInfo.key}@${matchInfo.pkg.version}`));
-    //
+
+    // info: specific version is enforced via a custom resolution
+    if (matchRef.hint === 'resolution') {
```

Comment:
I assume this is a foolproof way of checking if a package version was enforced by a `resolution` config? Seems to work :)